src/classes/QueryAggregateCommand.cls

```diff
@@ -0,0 +1,8 @@
+public enum QueryAggregateCommand {
```

Comment:
Same here - let's hold off on tackling new features like aggregates until we get your revisions wrapped up.